.github/workflows/bump-version-release.yml

```
@@ -234,6 +234,43 @@ jobs:
         if: steps.changed-files-map-template.outputs.any_changed == 'true'
         run: npm -w @mapsindoors/map-template publish --access public
 
+  components-docs-release:
+    name: Release Components Docs
+    needs: components-release-npm
+    runs-on: ubuntu-latest
+    if: ${{needs.components-release-npm.outputs.has_files_changed}} == 'true'
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)